api-reference/10 UI Widgets/dxDataGrid/1 Configuration/customizeExportData.md

```diff
@@ -1,11 +1,11 @@
 ---
 id: dxDataGrid.Options.customizeExportData
 type: function(columns, rows)
-deprecated: 
+deprecated: Since v20.1, we recommend using [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```

Comment:
```suggestion
deprecated: Since v20.1, we recommend [ExcelJS-based export](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/) which does not use this option.
```